pkg/controllers/localbuild/gitea.go

```diff
@@ -3,21 +3,32 @@ package localbuild
 import (
 	"context"
 	"embed"
+	"fmt"
 
 	"github.com/cnoe-io/idpbuilder/api/v1alpha1"
+	"github.com/cnoe-io/idpbuilder/pkg/util"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	ctrl "sigs.k8s.io/controller-runtime"
 )
 
 const (
-	giteaNamespace = "gitea"
-	// hardcoded secret name from what we have in the yaml installation file
+	// hardcoded values from what we have in the yaml installation file.
+	giteaNamespace   = "gitea"
 	giteaAdminSecret = "gitea-admin-secret"
+	// this is the URL accessible outside cluster. resolves to localhost
+	giteaIngressURL = "http://gitea.cnoe.localtest.me:8880"
```

Comment:
`8443` or `8880` for the port number?

Reply:
> Should be 8880. We should consider using TLS in the future, but it's all http for gitea as of now.

I don t think that we should append additional ports to the URL as we do now:
- `https://argocd.<host_IP>.<DOMAIN>:8443`, 
- `https://gitea.<host_IP>.<DOMAIN>:8880`, 
- `https://backstage.<host_IP>.<DOMAIN>:8443`,
- etc

Why: ?
- Adding more and more new ports will make the work of the users not really cool and easy as they will have to remember the ports to be used for each component
- Only default HTTP(S) ports are needed and will be recognized by the browser: 80, 443 automatically. 
- As the ingress controller which is acting as a [reverse proxy](https://en.wikipedia.org/wiki/Nginx) using nginx is able to forward from URL: `https://backend>.<host_ip>.<domain>` the packets to the target backends: argocd, backstage, vault, gitea, etc then adding new ports make the job to configure ingress + kind part of the idpbuilder hardder instead of having Ingress YAML files

Reply:
I agree from the user friendliness standpoint. The reason we are not using default http and https ports is that allocation of lower number port (< 1024) requires higher privileges that some organizations do not allow end users to have. I would like to use default ports but I would rather have the assurance that port allocation will work for all users. Maybe we can make it configurable in the future.